### PR TITLE
Plumb the Prow URL through to the SubmitQueue batch link

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Rename ProwURL to ProwUrl to match other URL options

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

Plumb the Prow URL through to the SubmitQueue batch link

The Prow URL was already being passed to the SubmitQueue as a
configuration option, but the link for batch merges was hard-coded to
use the k8s Prow instance. Now, the correct URL is plumbed through from
the configuration.

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

/cc @kargakis 

The first commit is optional, it just seemed reasonable to make `ProwUrl` match _e.g._  `ChartUrl` `HistoryUrl`...

### CODE REVIEW

File: mungegithub/mungers/submit-queue.go [hunk 1]
```diff
@@ -566,15 +568,15 @@ func (sq *SubmitQueue) RegisterOptions(opts *options.Options) sets.String {
 	opts.RegisterInt(&sq.AdminPort, "admin-port", 9999, "If non-zero, will serve administrative actions on this port.")
 	opts.RegisterString(&sq.Metadata.historyUrl, "history-url", "", "URL to access the submit-queue instance's health history.")
 	opts.RegisterString(&sq.Metadata.chartUrl, "chart-url", "", "URL to access the submit-queue instance's health charts.")
-	opts.RegisterString(&sq.ProwURL, "prow-url", "", "Prow data.json URL to read batch results")
+	opts.RegisterString(&sq.ProwUrl, "prow-url", "", "Prow base URL to read batch results and direct users to.")
```
Comment (rmmh) [inline mungegithub/mungers/submit-queue.go]: "Prow deployment base URL."

**Comment by rmmh** [comment]
Just a minor nit about the help text.

/lgtm
/joke

**Comment by stevekuznetsov** [comment]
@rmmh amended the flag helptext

**Pull request merged**